model/patch_lifecycle.go

```diff
@@ -300,7 +300,14 @@ func FinalizePatch(ctx context.Context, p *patch.Patch, requester string, github
 
 	_, err = thirdparty.GetCommitEvent(githubCtx, githubOauthToken, projectRef.Owner, projectRef.Repo, p.Githash)
 	if err != nil {
-		grip.Warning(errors.Wrap(err, "Couldn't fetch commit information").Error())
+		grip.Warning(message.Fields{
+			"error":    err,
```

Comment:
should use message.WrapError rather than putting it as a field (formats it better)